app/pages/news/index.vue

```diff
@@ -1,30 +1,33 @@
 <template>
-<<<<<<< HEAD
   <div class="container">
-    <input
-      v-model="search"
-      class="input"
-      type="search"
-      @input="onChange"
-      :placeholder="placeholder"
-    >
     <header class="flex flex-row items-center justify-between">
       <h1 class="mb-8">
```

Comment:
Acording to Csenge's design we changed the title block with black background-white letters
```
    <h3 class="title-block">
      News
    </h3>
```
see other main pages also. Maybe this got lost at the merge?